enclave_mgmt/compile_models.py

```diff
@@ -295,7 +289,6 @@ def create_models(output_path, all_raw_dfs):
 
     clean_raw_df = scrub_raw_dfs(all_raw_dfs)
 
-    demographics_df = demographics_model(clean_raw_df)
```

Comment:
We can remove the `demographics_model` function too now since it's dead code (this should also make codecov happy).